cmd/manager/main.go

```diff
@@ -67,10 +71,10 @@ func main() {
 	glog.Fatal(mgr.Start(signals.SetupSignalHandler()))
```

Comment:
This should be: 

    err := mgr.Start(signals.SetupSignalHandler())
    if err != nil {
        glog.Fatal(err)
    }

unless exiting from `Start` is always a fatal error.

Reply:
It is. Every time the manager stops, it needs to be restarted.

Reply:
Is restarting dependent on the exit code value?

Reply:
The point is to fail no matter what. I don't think the code matters. Tough, this is generated code.